sources/execDynawoAlgorithms.sh

```diff
@@ -75,6 +78,10 @@ setEnv() {
 
   # set LD_LIBRARY_PATH
   export LD_LIBRARY_PATH=$DYNAWO_ALGORITHMS_INSTALL_DIR/lib:$LD_LIBRARY_PATH
+
+  if [ -z "$MPIRUN_PATH" ]; then
+    MPIRUN_PATH="$DYNAWO_ALGORITHMS_THIRD_PARTY_INSTALL_DIR/mpich/bin/mpirun"
```

Comment:
path here is $DYNAWO_ALGORITHMS_THIRD_PARTY_INSTALL_DIR/bin/mpirun as it is a distrib

Reply:
OK